pysages/colvars/angles.py

```diff
@@ -128,6 +128,56 @@ def dihedral_angle(p1, p2, p3, p4):
     return np.arctan2(np.dot(np.cross(r, s), q), np.dot(r, s) * linalg.norm(q))
 
 
+class VectorAngle(FourPointCV):
+    """
+    Compute the angle formed by two vectors (four points).
+    Notice that the two vectors are defined as p1->p2, p3->p4.
+    """
+
+    @property
+    def function(self):
+        """
+        Returns
+        --------
+        Functions that calculates the angle value from a simulation snapshot.
+        """
+        return vector_angle
+
+
+def vector_angle(p1, p2, p3, p4):
```

Comment:
I think we should be able to reduce some code duplication by having our `angle` function taking four arguments and then change the `Angle` class call a lambda that calls `angle` with one of the arguments repeated.

Reply:
I see, that sounds cool